package-support.json

```diff
@@ -0,0 +1,16 @@
+{
+  "versions": [
+    {
+      "version": "*",
+      "target": {
+        "node": "supported"
+      },
+      "response": {
+        "type": "time-permitting"
+      },
+      "backing": {
+        "npm-funding": true
```

Comment:
Should this be changed to 

```
donations: ["https://opencollective.com/sequelize"]
```

?

Alternatively we should probably add a funding property to the package.json

Reply:
Yes, it's should! will work on it today. 😃

Reply:
Changed, also some changes for following the conventions from https://docs.npmjs.com/cli/v6/configuring-npm/package-json